VideoQuickStart/ViewController.swift

```diff
@@ -427,10 +442,14 @@ extension ViewController : RemoteParticipantDelegate {
         
         logMessage(messageText: "Unsubscribed from \(publication.trackName) video track for Participant \(participant.identity)")
 
-        if (self.remoteParticipant == participant) {
-            videoTrack.removeRenderer(self.remoteView!)
-            self.remoteView?.removeFromSuperview()
-            self.remoteView = nil
+        if self.remoteParticipant == participant {
+            cleanupRemoteParticipant()
+
+            // Find another Participant video to render, if possible.
+            var remainingParticipants = room?.remoteParticipants
+            let index = remainingParticipants?.index(of: participant)
+            remainingParticipants?.remove(at: index!)
+            renderRemoteParticipants(participants: remainingParticipants!)
```

Comment:
I am unsure about the filtering. The publication's track is already nil, so we could just include the Participant in the array. When a Participant with N video tracks is disconnecting, this also might result in momentarily subscribing to N-1 video tracks that are about to be unsubscribed.